app/assets/stylesheets/active_admin/print.scss

```diff
@@ -1,25 +1,29 @@
 /* Active Admin Print Stylesheet */
 
-// Set colors used elsewhere
+@import "active_admin/mixins/variables";
+
+$body-background-color: #fff;
+
+$font-family-sans-serif: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif !default;
+$font-size-base: 13px;
+$line-height-base: 1.5;
+
 $primary-color: black;
 $text-color: black;
 
-// Reset
-@import "active_admin/mixins/reset";
-@include global-reset;
+// Normalize
+@import "active_admin/normalize";
 
 // Partials
 @import "active_admin/typography";
 
 body {
-  font-family: Helvetica, Arial, sans-serif;
-  line-height: 150%;
-  font-size: 72%;
-  background: #fff;
-  width: 99%;
-  margin: 0;
-  padding: .5%;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.